creators/accra/trips_creator_accra.py

```diff
@@ -1,17 +1,16 @@
 # coding=utf-8
 
-import sys
-import json
-import re
 from datetime import timedelta, datetime
+
 from creators.trips_creator import TripsCreator
-from core.osm_routes import Route, RouteMaster
 
 
 class TripsCreatorAccra(TripsCreator):
+    ValueError = None
+    service_weekday = None
 
     def add_trips_to_schedule(self, schedule, data):
-        self.service_weekday = schedule.GetDefaultServicePeriod()
+        self.ValueError = schedule.GetDefaultServicePeriod()
```

Comment:
WTF ?

Reply:
Ups, thanks for noticing. Nasty copy and paste mistake I guess.